openmicroscopy/openmicroscopy
# Remove deprecated references to hudson.openmicroscopy.org.uk

OMERO.blitz API documentation still had old references to the OMERO job on the old CI infrastructure. Now the API documentation is included as part of the release and has a permanent URL, we use the latest API redirects to have non-broken links.

To test this PR,
- check the `See xxx.html` hyperlinks in the merge slice2html docs https://ci.openmicroscopy.org/job/OMERO-5.1-merge-build/ICE=3.5,jdk=8_LATEST,label=trout/javadoc/slice2html/index.html are correctly redirected to the latest released Javadoc API
- check the top-level banners of the Javadoc/slice2html documentation do not link to the deprecated OMERO job
